packages/kbn-apm-synthtrace-client/src/lib/infra/host.ts

```diff
@@ -20,17 +21,20 @@ interface HostDocument extends Fields {
   'host.ip'?: string;
   'host.os.name'?: string;
   'host.os.version'?: string;
+  'host.os.platform'?: string;
   'cloud.provider'?: string;
 }
 
 class Host extends Entity<HostDocument> {
-  cpu() {
+  cpu(cpuValue?: number) {
```

Comment:
(nit: ) I see 2 possible improvement here
1. Since this return HostMetrics with bunch of fields, you can make this parameter an object instead of string. This will allow you to extend this in future
2. Can we name this variable differently. Assigning `cpuValue` to `'system.cpu.total.norm.pct'` doesn't looks good. Something more related to this field will be better